dds/idl/anonymous.h

```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#ifndef anonymous_H
+#define anonymous_H
+
+#include "dds_generator.h"
+#include "utl_scoped_name.h"
+#include "ast.h"
```

Comment:
```suggestion

#include <utl_scoped_name.h>
#include <ast.h>
```

Since these are in another directory.